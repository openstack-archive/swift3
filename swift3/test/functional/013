#!/bin/bash

# List Multipart Uploads

. ./common


S3USER=tester

_s3_put /bucket

upload_id=$(_s3_post /bucket/%e3%81%82%2etxt?uploads _xq /InitiateMultipartUploadResult/UploadId)
upload_id=$(_s3_post /bucket/%e3%81%84%2etxt?uploads _xq /InitiateMultipartUploadResult/UploadId)
upload_id=$(_s3_post /bucket/object?uploads _xq /InitiateMultipartUploadResult/UploadId)

_s3_get /bucket?uploads -D - | _filter_curl xml

