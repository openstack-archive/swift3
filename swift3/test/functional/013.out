QA output created by 013
> s3curl --id tester -- -X PUT http://SWIFT_HOST/bucket... 200
> s3curl --id tester -- -X POST _xq /InitiateMultipartUploadResult/UploadId http://SWIFT_HOST/bucket/%e3%81%82%2etxt?uploads... 200
> s3curl --id tester -- -X POST _xq /InitiateMultipartUploadResult/UploadId http://SWIFT_HOST/bucket/%e3%81%84%2etxt?uploads... 200
> s3curl --id tester -- -X POST _xq /InitiateMultipartUploadResult/UploadId http://SWIFT_HOST/bucket/object?uploads... 200
> s3curl --id tester -- -X GET -D - http://SWIFT_HOST/bucket?uploads... 200
HTTP/1.1 200 OK
Content-Length: LENGTH
Content-Type: application/xml
Date: DATE
x-amz-id-2: TXID
x-amz-request-id: TXID
X-Trans-Id: TXID

<?xml version='1.0' encoding='UTF-8'?>
<ListMultipartUploadsResult xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
  <Bucket>bucket</Bucket>
  <KeyMarker/>
  <UploadIdMarker/>
  <NextKeyMarker>い.txt</NextKeyMarker>
  <NextUploadIdMarker>UPLOAD_ID</NextUploadIdMarker>
  <MaxUploads>1000</MaxUploads>
  <IsTruncated>false</IsTruncated>
  <Upload>
    <Key>object</Key>
    <UploadId>UPLOAD_ID</UploadId>
    <Initiator>
      <ID>TESTER</ID>
      <DisplayName>TESTER</DisplayName>
    </Initiator>
    <Owner>
      <ID>TESTER</ID>
      <DisplayName>TESTER</DisplayName>
    </Owner>
    <StorageClass>STANDARD</StorageClass>
    <Initiated>DATE</Initiated>
  </Upload>
  <Upload>
    <Key>あ.txt</Key>
    <UploadId>UPLOAD_ID</UploadId>
    <Initiator>
      <ID>TESTER</ID>
      <DisplayName>TESTER</DisplayName>
    </Initiator>
    <Owner>
      <ID>TESTER</ID>
      <DisplayName>TESTER</DisplayName>
    </Owner>
    <StorageClass>STANDARD</StorageClass>
    <Initiated>DATE</Initiated>
  </Upload>
  <Upload>
    <Key>い.txt</Key>
    <UploadId>UPLOAD_ID</UploadId>
    <Initiator>
      <ID>TESTER</ID>
      <DisplayName>TESTER</DisplayName>
    </Initiator>
    <Owner>
      <ID>TESTER</ID>
      <DisplayName>TESTER</DisplayName>
    </Owner>
    <StorageClass>STANDARD</StorageClass>
    <Initiated>DATE</Initiated>
  </Upload>
</ListMultipartUploadsResult>
